.vscode/settings.json

```diff
@@ -115,120 +42,239 @@
+      "type": "text"
     },
     {
+      "default": false,
       "label": "Breaking change",
       "name": "breaking_change",
       "type": "boolean",
-      "value": "BREAKING CHANGE: ",
-      "default": false
+      "value": "BREAKING CHANGE: "
     },
     {
+      "description": "Optional footer",
       "label": "Footer",
+      "multiline": true,
       "name": "footer",
-      "description": "Optional footer",
-      "type": "text",
-      "multiline": true
+      "type": "text"
     }
   ],
+  "dependi.npm.lockFileEnabled": true,
+  "editor.formatOnSave": true,
+  "editor.rulers": [
+    {
+      "color": "#cc999966",
+      "column": 100
+    },
+    {
+      "color": "#aaaaaa66",
+      "column": 80
+    },
+  ],
+  "eslint.codeAction.showDocumentation": {
+    "enable": true
+  },
+  "eslint.codeActionsOnSave.mode": "all",
+  "eslint.enable": true,
+  "eslint.lintTask.enable": true,
+  "eslint.onIgnoredFiles": "warn",
+  "eslint.options": {
+    "overrideConfigFile": "./eslint.config.js"
+  },
+  "eslint.problems.shortenToSingleLine": true,
+  "eslint.useFlatConfig": true,
+  "eslint.runtime": "node",
+  "eslint.validate": [
+    "javascript",
+    "typescript",
+    "json",
+    "jsonc"
+  ],
+  "eslint.workingDirectories": [
+    {
+      "pattern": "."
+    }
+  ],
+  "explorer.excludeGitIgnore": true,
+  "files.exclude": {
+    "**/.DS_Store": true,
+    "**/.git": true,
+    "**/dist": true,
+    "**/node_modules": true,
+    "tracerbench-results": true
+  },
+  "files.insertFinalNewline": true,
+  "files.trimTrailingWhitespace": true,
+  "files.watcherExclude": {
+    "**/.git/objects/**": true,
+    "**/.git/subtree-cache/**": true
+  },
+  "inline-bookmarks.expert.custom.styles": {
+    "active": {
+      "dark": {
+        "color": "#00ffff",
+        "fontWeight": "bold"
+      },
+      // teal
+      "gutterIconColor": "#009999",
+      "light": {
+        "color": "#009999",
+        "fontWeight": "bold"
+      },
+      "overviewRulerColor": "rgba(0, 150, 150, 0.7)"
+    },
+    "fixme": {
+      "dark": {
+        "color": "#ff0000",
+        "fontWeight": "bold"
+      },
+      // yellow
+      "gutterIconColor": "#990000",
+      "light": {
+        "color": "#990000",
+        "fontWeight": "bold"
+      },
+      "overviewRulerColor": "rgba(150, 0, 0, 0.7)"
+    },
+    "warn": {
+      "dark": {
+        "color": "#ffff00",
+        "fontWeight": "bold"
+      },
+      // yellow
+      "gutterIconColor": "#999900",
+      "light": {
+        "color": "#999900",
+        "fontWeight": "bold"
+      },
+      "overviewRulerColor": "rgba(150, 150, 0, 0.7)"
+    }
+  },
+  "inline-bookmarks.expert.custom.words.mapping": {
+    "audit": [
+      "@audit(\\s|$)"
+    ],
+    "fixme": [
+      "@fixme(\\s|$)"
+    ],
+    "warn": [
+      "@premerge(\\s|$)"
+    ]
+  },
+  "inline-bookmarks.view.showVisibleFilesOnly": true,
+  "javascript.preferences.importModuleSpecifier": "project-relative",
```

Comment:
auto-import from other workspace packages directly (i.e. `from "@glimmer/vm"`) rather than importing
(incorrectly) from a relative path (which would escape the package boundary).